clouddriver-api/src/main/java/com/netflix/spinnaker/clouddriver/security/AccountCredentials.java

```diff
@@ -98,43 +96,18 @@ default String getAccountId() {
    */
   String getCloudProvider();
 
+  default boolean isEnabled() {
+    return true;
+  }
+
   /**
    * A user in ANY required group should be allowed access to this account.
    *
    * @return the group names that govern access to this account, empty indicates a public account
    *     accessible by all.
    */
   @Deprecated
-  List<String> getRequiredGroupMembership();
-
-  default Permissions getPermissions() {
```

Comment:
Where'd this stuff go?

Reply:
I left `getRequiredGroupMembership` out of the interface since it is deprecated. `getPermissions` can't be in api because it requires fiat as part of the contract. It is possible to write a plugin with this interface. And then this membership/permission stuff can be redone separately.